test/AutoDiff/stdlib/tgmath_derivatives.swift.gyb

```diff
@@ -114,6 +137,43 @@ DerivativeTests.test("${op}_${T}") {
 %end
     }
   }
+
+  // pow
+  let eps:${T} = 0.01
+  let ulps:${T} = eps/eps.ulp
+  for a in -3...3 {
```

Comment:
I think the logic would be easier to read if this was split into 2 or 3 separate loops, even if it costs a bit of code duplication in the assertions.

```
// Check derivatives for negative base.
for a in -3..<0 {
  for b in -3...3 {
    ...
  }
}

// Check derivatives for base 0.
for b in -3...3 {
  ...
}

// Check derivatives for positive base.
...
```

Reply:
I did split this logic into 3 loops and it is more code but it reads better, thanks for the remark.